deploy/00_multisig.ts

```diff
@@ -5,21 +5,28 @@ import { DAY } from "../test-ts/utils";
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deploy } = hre.deployments;
 
-  const { deployer, multisigOwner0, multisigOwner1, multisigOwner2 } = await hre.getNamedAccounts();
-  const owners = [multisigOwner0, multisigOwner1, multisigOwner2];
+  // Get a list the named accounts based on network
+  const namedAccounts = await hre.getNamedAccounts();
+
+  const deployer = namedAccounts.deployer;
+  const minDelay = Number(process.env.TIME_LOCK_MIN_DELAY);
+  const delay = Number(process.env.TIME_LOCK_DELAY);
```

Comment:
we read these from .env as they could not be stored in hardhat.config.ts

Reply:
Why not? Could the config be the one place where we read from .env, and then use the config in deployment scripts? In particular, defaults could be set in the config, and then .env files could be used to override them. Thoughts?

Reply:
Same question i had. The config is of type `HardhatUserConfig` and does not seem to have provision for keeping other things aside the the defined fields.
Its fully opinionated .

Reply:
Does the config need to be of that type?

Reply:
Thanks for your feedbacks @m-chrzan 
Yes apparently the args are strongly typed.